src/NavBar.jsx

```diff
@@ -12,7 +12,7 @@ const MenuContainer = styled.div`
 
 const Nav = styled.nav`
   display: flex;
-  max-height: 3rem;
+  max-height: 2rem;
```

Comment:
This keeps changing every PR :laughing:

Reply:
Mario changed that :stuck_out_tongue_closed_eyes:

Reply:
Yeah, while working on the design the navbar was too big so we modified it a bit